qiskit/circuit/quantumcircuit.py

```diff
@@ -2406,24 +2406,24 @@ def remove_final_measurements(self, inplace: bool = True) -> Optional["QuantumCi
 
     @staticmethod
     def from_qasm_file(path: str) -> "QuantumCircuit":
-        """Take in a QASM file and generate a QuantumCircuit object.
+        """Take in a OpenQASM2 file and generate a QuantumCircuit object.
```

Comment:
"a OpenQASM2" should be "an OpenQASM2". Elsewhere in Qiskit (include a change above) "an OpenQASM xx" is used.

Reply:
In fact, it should probably be "an OpenQASM 2.0 program" since you use "OpenQASM 2.0" above. (By typical hyphenation rules it should even be "an OpenQASM-2.0 program".